libraries/utils/src/render-lines-on-sp.ts

```diff
@@ -1,31 +1,27 @@
 import * as DG from 'datagrok-api/dg';
 import { Subject } from 'rxjs';
-
-export enum LineDirection {
-    pointToLowerValue = 'lower',
-    pointToHigherValue = 'higher',
-}
-
-export type ILineOpts = {
-    points: Uint32Array;
-    id: number;
-    color?: string;
-    directionCol?: string
-    direction?: LineDirection;
-    opacity?: number;
-    width?: number;
+import BitArray from './bit-array';
+
+export type ILines = {
+    from: Uint32Array;
+    to: Uint32Array;
+    colors?: string[];
```

Comment:
Typically, you would have one color and one width per series, as well as other parameters (like drawing arrows, etc). So I suggest to do the following:

```
  color?: string;       // common color. Use [colors] if you need individual colors per line
  colors?: string[];   // line colors. If they are the same for the series, use [color] instead.
```

Reply:
Done